src/Dialect/Krnl/Krnl.td

```diff
@@ -86,7 +86,7 @@ def KrnlCallOp : Op<Krnl_Dialect, "call",
       // Should mlir::OpTrait::AttrSizedOperandSegments be used?
       // The case for krnl.Call is simple.
       DefaultValuedAttr<SI64Attr, "1">:$numOfOutput,
-      Variadic<AnyType>:$parameters);
+      Variadic<AnyTypeOf<[AnyType, NoneType]>>:$parameters);
```

Comment:
Have you tried to not add NoneType here? I am not 100% sure whether AnyType includes NoneType.

Reply:
I have tried the original Krnl.td, and have confirmed that it works for lit tests.
Accordingly I use the original Krnl.td in this PR. Thanks for your variable comments!